server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserMailboxesRoutesTest.java

```diff
@@ -90,6 +90,8 @@
 class UserMailboxesRoutesTest {
```

Comment:
Can we add a test for the following mailboxes: 

`.` `ends.with.dot.` `.starts.with.dot` ?

Reply:
should be another cases

Reply:
The ticket title is : `[WebAdmin] improve mailbox localpart validate`, I think those cases fit in :)